src/lib/routes/admin-api/api-token.ts

```diff
@@ -168,7 +168,7 @@ export class ApiTokenController extends Controller {
             201,
             res,
             apiTokenSchema.$id,
-            serializeDates(token),
+            serializeDates({ ...token, tokenName: token.username }),
```

Comment:
Should the `tokenName` be added by the service/store when you `createApiToken`? That way, you don't need to remember to map it manually? If we don't want to store it in the database, you could still use the `fromRow` method (that I assume exists in the store) to populate it.

Reply:
Thank you, this was one of the things I specifically wanted feedback on! Is putting this in the model closer to DB when it's not backed by the DB schema something to be cautious about?

Reply:
Ah, that's fair. To be honest, I don't know what the pattern we prefer to use for this is, but my thinking is that if tha mapping is done in the store, then nowhere else needs to worry about it. And even if the backing store doesn't change the column names, the store interface still acts as if it has changed. but you'd need to ask someone else about whether that's a pattern we follow in this code base or not.

Reply:
> if tha mapping is done in the store, then nowhere else needs to worry about it

Yes please! I don't see a problem with this approach and it makes this much less risky. I hate touching token things

Reply:
I concur.